documents/api/viewsets.py

```diff
@@ -57,6 +63,8 @@ class DocumentMetadataViewSet(AuditLoggingModelViewSet):
 
     def get_queryset(self):
         user = self.request.user
+        if user.is_anonymous:
```

Comment:
This could be done be IsAuthenticated permissions?

Reply:
I used this solution as it matches the code style elsewhere in the project.